src/components/studies/StudyBuilder.tsx

```diff
@@ -138,6 +139,29 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
   const studyDataUpdateFn = useStudyInfoDataDispatch()
   const [open, setOpen] = React.useState(true)
   const [displayBanner, setDisplayBanner] = React.useState(false)
+  const [bannerType, setBannerType] = React.useState('')
+
+  React.useEffect(() => {
+    setBannerType(getBannerType())
+    if (getBannerType() !== 'success' && getBannerType() !== 'error') {
+      setDisplayBanner(true)
+    }
+  }, [])
+
+  const getBannerType = () => {
+    switch (builderInfo?.study?.phase) {
+      case 'in_flight':
+        return 'live'
+      case 'withdrawn':
+        return 'withdrawn'
+      case 'analysis':
+      case 'completed':
+        return 'completed'
+      default:
+        const errors = section === 'scheduler' ? schedulerErrors : error
+        return errors.length > 0 ? 'error' : 'success'
+    }
+  }
```

Comment:
can we change it to take the phase (StudyPhase  type)  and section to make it more explicit rather than just using what's available in the component. It makes it more explicit what it depends on ./ We can also put it above the component